<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LugarFiltros</class>
 <widget class="QDialog" name="LugarFiltros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 195, 221);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(203, 195, 221);</string>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Pictures/turr.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_atras">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Atrás</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tema">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/bigfood.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_tematexto">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>421</width>
     <height>118</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jamrul</family>
     <pointsize>36</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 36pt &quot;Jamrul&quot;;
color: rgb(73, 60, 107);</string>
   </property>
   <property name="text">
    <string>COMIDA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>71</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/iconoprecio.png</pixmap>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_precio">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(221, 212, 255);
selection-color: rgb(92, 53, 102);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>379</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconocalif.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Calificación</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_estrella">
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(221, 212, 255);
selection-color: rgb(92, 53, 102);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>379</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/iconoubicacion.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ubicación</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_lugar">
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(221, 212, 255);
selection-color: rgb(92, 53, 102);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>51</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
